packages/ButtonGroup/src/ButtonGroup.styles.js

```diff
@@ -0,0 +1,32 @@
+import tokens from "@paprika/tokens";
+import styled from "styled-components";
+import ButtonItem from "./components/ButtonItem";
+
+const fullWithStyles = `
+  display: flex;
+  width: 100%;
+`;
+
+export const Button = styled(ButtonItem)``;
+
+export const ButtonGroup = styled.div`
+  display: inline-flex;
+
+  ${Button} {
+    border-radius: 0;
+
+    &:first-child {
+      border-radius: ${tokens.spaceSm} 0 0 ${tokens.spaceSm};
```

Comment:
Even though the result is the same, it would be better to use the more semantic token `button.borderRadius` here.